alf/examples/sac_bipedal_walker.gin

```diff
@@ -36,18 +30,13 @@ SacAlgorithm.alpha_optimizer=@alpha/Adam()
 SacAlgorithm.target_update_tau=0.005
 OneStepTDLoss.td_error_loss_fn=@losses.element_wise_squared_loss
 
-Agent.action_spec = %action_spec
-Agent.rl_algorithm_cls=@SacAlgorithm
-
 # training config
-TrainerConfig.trainer=@sync_off_policy_trainer
 TrainerConfig.initial_collect_steps=32000
 TrainerConfig.mini_batch_length=4
 TrainerConfig.unroll_length=1
 TrainerConfig.mini_batch_size=4096
 TrainerConfig.clear_replay_buffer=False
 TrainerConfig.num_updates_per_train_step=1
-TrainerConfig.algorithm_ctor=@Agent
 TrainerConfig.num_iterations=200000
 TrainerConfig.checkpoint_interval=100000
```

Comment:
this is removed. now is num_checkpoints

Reply:
I know. This PR is just to include 'sac.gin' for all related gin files. I guess sac_bipedal_walker.gin hasn't been reproduced by anyone, so the configuration is untouched.

Reply:
In this case, it might be better to keep these gin files as is until they are reproduced. Then a shared ```sac.gin``` might be introduced. 
This leads to a cleaner versioning of these files.

Reply:
Reverted some files. Was trying to save some hassles of making the change for the future.